HunterPie/App.xaml.cs

```diff
@@ -156,8 +156,8 @@ private void OnProcessFound(object sender, ProcessManagerEventArgs e)
             _richPresence = DiscordPresenceController.GetPresenceBy(context);
 
             WidgetManager.Hook(context);
-            GamePatchers.Run(context);
-            ContextInitializers.Initialize(context);
+            // Start initialization but don't wait for it.
+            _ = ContextInitializers.InitializeAsync(context);
```

Comment:
Pretty sure we should `await` this, passing a not initialized `Context` to the overlay (and to plugins in the future) will probably cause issues eventually.